src/Apps/Artist/Routes/Shows/ArtistShows.tsx

```diff
@@ -84,76 +90,83 @@ class ArtistShows extends Component<ArtistShowsProps, LoadingAreaState> {
-            <Row>
-              <Col>
-                <Row>
-                  <Col>
-                    <LoadingArea isLoading={this.state.isLoading}>
-                      {this.props.status === "running" ? (
-                        <ShowBlocks flexDirection={blockDirection} flexWrap>
-                          {this.props.artist.showsConnection.edges.map(
-                            ({ node }, edgeKey) => {
-                              return (
-                                <ArtistShowBlockItem
-                                  key={edgeKey}
-                                  blockWidth={blockWidth}
-                                  imageUrl={node.cover_image.cropped.url}
-                                  partner={node.partner.name}
-                                  name={node.name}
-                                  exhibitionInfo={node.exhibition_period}
-                                  pr={pr}
-                                  pb={pb}
-                                  href={node.href}
-                                  city={node.city}
-                                />
-                              )
-                            }
-                          )}
-                        </ShowBlocks>
-                      ) : (
-                        <ShowList>
-                          {this.props.artist.showsConnection.edges.map(
-                            ({ node }, edgeKey) => {
-                              return (
-                                <ArtistShowListItem
-                                  key={edgeKey}
-                                  city={node.city}
-                                  partner={node.partner.name}
-                                  name={node.name}
-                                  exhibitionInfo={node.exhibition_period}
-                                  href={node.href}
-                                />
-                              )
-                            }
-                          )}
-                        </ShowList>
-                      )}
-                    </LoadingArea>
-                  </Col>
-                </Row>
-
-                {this.props.status === "running" && (
-                  <Box py={2}>
-                    <Separator />
-                  </Box>
-                )}
-
-                <Row>
-                  <Col>
-                    <Flex justifyContent="flex-end">
-                      <PaginationFragmentContainer
-                        pageCursors={
-                          this.props.artist.showsConnection.pageCursors as any
-                        }
-                        onClick={this.loadAfter}
-                        onNext={this.loadNext}
-                        scrollTo={this.props.scrollTo}
-                      />
-                    </Flex>
-                  </Col>
-                </Row>
-              </Col>
-            </Row>
+            <React.Fragment>
+              <ShowDivider />
```

Comment:
This is getting a little iffy. The 'layout' component one level up needs to know if this data exists, in order to include section headers, spacers/dividers, etc.

However, it doesn't know that, so had to move that here. But, if I move the 'scroll to' position into here as well, that seemed to not feel as nice scroll-wise.

I was wondering about this before, how to compose components when they do need to know info about each other. Seems like the React/component paradigm can be rigid when it comes to this...not sure the best way to solve. I think there's still an errant divider line floating around.